ingestion_service/producer/core/routers/api_endpoints_input.py

```diff
@@ -0,0 +1,159 @@
+                                                                 "Cognitive therapy"]
+                                              },
+                                              {
+                                                  "disorder": "Parkinson's disease",
+                                                  "description": "A neurodegenerative disorder characterized by tremors, rigidity, and difficulty with movement.",
+                                                  "symptoms": ["Tremors", "Bradykinesia", "Postural instability"],
+                                                  "treatments": ["Levodopa", "Deep brain stimulation (DBS)"]
+                                              },
+                                              {
+                                                  "disorder": "Schizophrenia",
+                                                  "description": "A chronic and severe mental disorder that affects how a person thinks, feels, and behaves.",
+                                                  "symptoms": ["Hallucinations", "Delusions", "Disorganized thinking"],
+                                                  "treatments": ["Antipsychotic medications", "Psychotherapy"]
+                                              }
+                                          ]
+
+                                      }
+                                  }
+                              ],
+                          ),
+                      ], ):
+    try:
+        main_model_schema = jsoninput.json()
+        publish_message(main_model_schema)
+    except json.JSONDecodeError as e:
+        raise HTTPException(status_code=400, detail="Invalid JSON" + str(e))
+
+    return JSONResponse(content={"message": f"Data uploaded successfully {main_model_schema}"})
+
+
+@router.post("/ingest/raw/jsonld")
+async def ingest_json(user: Annotated[LoginUserIn, Depends(get_current_user)],
+                      jsonldinput: Annotated[
+                          InputJSONSLdchema,
+                          Body(
+                              examples=[
+                                  {
+                                      "type": "jsonld",
+                                      "kg_data": {"@context": "https://schema.org", "@type": "Person",
+                                                  "name": "John Doe"}
+                                  }
+                              ],
+                          ),
+                      ], ):
+    try:
+
+        json_data = jsonldinput.json()
+        if is_valid_jsonld(json_data):
+            publish_message(json_data)
+            return JSONResponse(content={"message": "Data uploaded successfully"})
+        else:
+            return JSONResponse(content={"message": "Invalid format data! Please provide correct JSON-LD data."})
+
+    except json.JSONDecodeError as e:
+        raise HTTPException(status_code=400, detail="Invalid JSON" + str(e))
```

Comment:
These seem to be duplicative -- both scenarios seem to reject an invalid JSON-LD submission

Reply:
One is for JSON and other is for JSON-LD as indicated in the message.